src/main.c

```diff
@@ -908,7 +908,7 @@ scrot_grab_stack_windows(void)
     Bool delete      = False;
     int actual_format_return;
     Atom actual_type_return;
-    Scrot_Imlib_List *list_images   = NULL;
+    ScrotImlibList *list_images   = NULL;
```

Comment:
See [Pointers and References](https://webkit.org/code-style-guidelines/#pointers-and-references)

Reply:
This is expected and will be converted when the effort comes to the main.c file. 

I intentionally left it as is. For now, only the renaming part matters. :-)